admin_guide/ipsec.adoc

```diff
@@ -250,6 +250,11 @@ and begin encrypting:
 # systemctl enable ipsec
 ----
 
+[NOTE]
```

Comment:
I think this duplicates what's already in paragraph 2 of the pre-req section https://github.com/openshift/openshift-docs/blob/master/admin_guide/ipsec.adoc#step-1-prerequisites